.circleci/config.yml

```diff
@@ -104,7 +104,7 @@ jobs:
 
       - slack/notify:
           message: "Orbs from
-             *<https://github.com/${CIRCLE_PROJECT_USERNAME}/${CIRCLE_PROJECT_REPONAME}/commit/${CIRCLE_SHA1}|commit ${CIRCLE_SHA1}>*
+             *<https://github.com/${CIRCLE_PROJECT_USERNAME}/${CIRCLE_PROJECT_REPONAME}/commit/${CIRCLE_SHA1}|commit ${CIRCLE_SHA1:0:7}>*
              on the *<https://github.com/${CIRCLE_PROJECT_USERNAME}/${CIRCLE_PROJECT_REPONAME}/tree/${CIRCLE_BRANCH}|${CIRCLE_BRANCH} branch>*
```

Comment:
Oh I wanted to ask about this! I noticed for docker-publish that the entire sha is used as the tag, see here -> https://hub.docker.com/r/pydicom/dicom-ocr-cleaner/tags. For some of the builds I've done, I use the first 10 characters of the sha, and based on the [Git book](https://git-scm.com/book/en/v2/Git-Tools-Revision-Selection#Short-SHA-1) they say:

> Generally, eight to ten characters are more than enough to be unique within a project.

Is it okay to use the minimum, or is it worth it to be conservative and go for 10? Does it hurt? A docker (minified) sha is 12, for reference.